content-services/latest/release/index.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Release Notes
```

Comment:
I think the title should be "what's new". This is what we have used before, and what customers will look for.